python/examples/pylammps/elastic/elastic.py

```diff
@@ -0,0 +1,309 @@
+    c2pos = lmp.variables["d2"].value
+    c3pos = lmp.variables["d3"].value
+    c4pos = lmp.variables["d4"].value
+    c5pos = lmp.variables["d5"].value
+    c6pos = lmp.variables["d6"].value
+
+    # Combine posiive and negative
+    lmp.variable("C1{} equal {}".format(idir, 0.5*(c1neg+c1pos)))
+    lmp.variable("C2{} equal {}".format(idir, 0.5*(c2neg+c2pos)))
+    lmp.variable("C3{} equal {}".format(idir, 0.5*(c3neg+c3pos)))
+    lmp.variable("C4{} equal {}".format(idir, 0.5*(c4neg+c4pos)))
+    lmp.variable("C5{} equal {}".format(idir, 0.5*(c5neg+c5pos)))
+    lmp.variable("C6{} equal {}".format(idir, 0.5*(c6neg+c6pos)))
+
+    return
+
+def elastic():
+    """ Compute elastic constant tensor for a crystal
+
+     In order to calculate the elastic constants correctly, care must be taken to specify
+     the correct units (units). It is also  important to verify that the minimization of energy
+     w.r.t atom  positions in the deformed cell is fully converged.
+     One indication of this is that the elastic constants are insensitive
+     to the choice of the variable ${up}. Another is to check
+     the final max and two-norm forces reported in the log file. If you know
+     that minimization is not required, you can set maxiter = 0.0 """
+
+    parser = ArgumentParser(description='A python script to compute elastic properties of bulk materials')
+
+    parser.add_argument("input_data_file", help="The full path & name of the lammps data file.")
+    parser.add_argument("kim_model", help="the KIM ID of the interatomic model archived in OpenKIM")
+    parser.add_argument("elements", nargs='+', default=['Au'], help="a list of N chemical species, which defines a mapping between atom types in LAMMPS to the available species in the OpenKIM model")
+    parser.add_argument("--min_style", default="cg", help="which algorithm will be used for minimization from lammps")
+    parser.add_argument("--minimize", type=float, nargs=4, default=[1.0e-4, 1.0e-6, 100, 1000], help="minimization parameters")
+    parser.add_argument("--up", type=float, default=1.0e-6, help="the deformation magnitude (in strain units)")
+    args = parser.parse_args()
+
+    L = PyLammps()
+
+    L.units("metal")
+
+    # Define the finite deformation size.
+    #Try several values to verify that results do not depend on it.
+    L.variable("up equal {}".format(args.up))
+
+    # Define the amount of random jiggle for atoms. It prevents atoms from staying on saddle points
+    atomjiggle = 1.0e-5
+
+    # metal units, elastic constants in GPa
+    cfac = 1.0e-4
+
+    # Define minimization parameters
+    L.variable("dmax equal 1.0e-2")
+
+    L.boundary("p", "p", "p") # periodic boundary conditions in all three directions
+    L.box("tilt large") # to avoid termination if the final simulation box has a high tilt factor
+
+    # use the OpenKIM model to set the energy interactions
+    L.kim_init(args.kim_model, "metal", "unit_conversion_mode")
```

Comment:
This fails with `ERROR: This command is no longer available (src/deprecated.cpp:43)`. This should now be
```suggestion
    L.kim("init", args.kim_model, "metal", "unit_conversion_mode")
```

Reply:
thanks for fixing that. it seems i am using an old version of the interface

Reply:
@evoyiatzis that's just one of many, basically all `kim_xyz` commands are now `kim xyz`, so the rest still needs to be updated.

Reply:
@rbberger I will not update my local installation in the next weeks. I can make some changes following the manual but without testing it does not make a lot of sense. should i close the request or leave it hanging for some time?